lib/helpers/utils.js

```diff
@@ -142,6 +142,12 @@ export function shouldFetchLicense() {
   );
 }
 
+export function shouldFetchVCS() {
```

Comment:
Shall we get rid of this and enable this as part of FETCH_LICENSE?

Reply:
I suggest we leave it separately. License information is collected for a long time on large projects (each dependency = request to pkg.go.dev). In the case of vcs, a request to pkg.go.dev is sent only for packages that do not start with github.com/gitlab.com (and this is 90% of the packages in my case). So i use only GO_FETCH_VCS without FETCH_LICENSE